src/pages/projects.jsx

```diff
@@ -15,48 +15,86 @@ const StyledH1 = styled.h1`
     margin-bottom: 20px;
 `;
 
+const StyledP1 = styled.p`
+    font-size: 25px;
+    margin-bottom: 100px;
+`;
+
 const StyledP = styled.p`
     font-size: 25px;
+    margin-bottom: 20px;
 `;
 
 const StyledImg = styled.img`
     border-style: solid;
     border-color: black;
-    margin-bottom: 30px;
+    margin-bottom: 80px;
+`;
+
+const StyledButt = styled.button`
+    border: 3px solid #048c24;
+    border-radius: 100px;
+    background-color: #90fca9;
+    color: #048c24;
+    padding: 20px 40px;
+    transition-duration: 1s;
+    transition-property: font-size;
+    font-size: ${(props) => props.inputFont}px;
+    margin-top: 100px;
+    margin-bottom: 100px;
+    margin-left: auto;
+    margin-right: auto;
+    display: block;
 `;
 
+let inc = true;
+
 function Projects() {
+    const [size, setSize] = useState(20);
+    let change = 20;
+    let biggerSmaller = "BREATHE IN";

```

Comment:
It is unclear what this variable is meant to be used for just by looking at the name. Can you name it something that represents what it is used for, like `buttonText`?